### DISCUSSION TITLE
How is the `Query` parameter bound in the `OnGetAsync` methods?

### DISCUSSION THREAD

**Discussion by hlaueriksson** [jbogard/ContosoUniversityDotNetCore-Pages#63] [Q&A] [answered]
The `Query` parameter in the `OnGetAsync` methods, for example in [Courses/Delete.cshtml.cs](https://github.com/jbogard/ContosoUniversityDotNetCore-Pages/blob/bff436873124954cb79333afe6a19fdd5e3cf94d/ContosoUniversity/Pages/Courses/Delete.cshtml.cs#L26), is automatically bound.

Is this configured in the project, or is it something that comes out of the box from Razor Pages?

**Comment by jbogard** [comment, marked as answer]
Yep! It comes out-of-the-box with Razor Pages and model binding.
